pkg/autodiscovery/integration/config.go

```diff
@@ -394,3 +395,26 @@ func (c *Config) Digest() string {
 
 	return strconv.FormatUint(h.Sum64(), 16)
 }
+
+// FastDigest returns an hash value representing the data stored in this configuration.
+// Difference with Digest is that FastDigest does not consider that difference may appear inside Instances
+// allowing to remove costly YAML Marshal/UnMarshal operations
+// The ClusterCheck field is intentionally left out to keep a stable digest
+// between the cluster-agent and the node-agents
+func (c *Config) FastDigest() uint64 {
+	h := murmur3.New64()
+	h.Write([]byte(c.Name)) //nolint:errcheck
+	for _, i := range c.Instances {
+		h.Write([]byte(i))
+	}
+	h.Write([]byte(c.InitConfig)) //nolint:errcheck
+	for _, i := range c.ADIdentifiers {
+		h.Write([]byte(i)) //nolint:errcheck
+	}
+	h.Write([]byte(c.NodeName))                                    //nolint:errcheck
```

Comment:
To avoid the `//nolint:errcheck` 
```suggestion
	h.Write([]byte(c.NodeName))
```

Reply:
(Assuming `h.Write` never returns a non-nil error, otherwise we should check for it)